services/tenant-ui/frontend/src/store/holderStore.ts

```diff
@@ -35,6 +36,36 @@ export const useHolderStore = defineStore('holder', () => {
     );
   }
 
+  const tenantApi = useTenantApi();
+
+  async function acceptCredentialOffer(cred_id: string) {
+    await tenantApi
```

Comment:
Same with these functions - don't match up with similar existing calls... 
Did we decide to change the pattern up? Cool if we have, but I don't see other stores having been updated. 🤷